teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,6 @@
 try:
-    import pycodestyle as pep8
+    from flake8.formatting import base  # noqa
 except ImportError:
-    import pep8
-
-import re
-
-from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
-
-
-name = 'teamcity'
-version = __version__
-enable_teamcity = is_running_under_teamcity()
```

Comment:
because of this line --teamcity option is actually not required. Could we reproduce this in v3?
(I want to run flake8 without any parameters and still get TeamCity output under TeamCity build)